libmscore/layout.cpp

```diff
@@ -2034,6 +2034,7 @@ void Score::createMMRest(Measure* m, Measure* lm, const Fraction& len)
                         if (!nks) {
                               nks = ks->generated() ? ks->clone() : toKeySig(ks->linkedClone());
                               nks->setParent(ns);
+                              nks->setGenerated(true);
```

Comment:
So, the policy is, elements in mmrests should always be considered generated, because we do literally generate them from the underlying measure?  I'm not sure we do that consistently, maybe we should?